app/models/github_reckoner.rb

```diff
@@ -1,27 +0,0 @@
-class GithubReckoner < Struct.new(:user)
```

Comment:
So happy we can get rid of the GithubReckoner. I clearly should do a better job of reading the docs. Thank you, Jessie!